standalone/src/main/java/io/pravega/local/LocalPravegaEmulator.java

```diff
@@ -53,7 +53,7 @@ public LocalPravegaEmulator build() {
                     .containerCount(4)
                     .restServerPort(restServerPort)
                     .enableRestServer(enableRestServer)
-                    .enableMetrics(false)
+                    .enableMetrics(true)
```

Comment:
If we enable metrics in Pravega standalone, maybe we will need a test to verify that metrics are working on standalone as well.

Reply:
Thank you, I enabled it in local test but forgot to revert it back. Fixed.
(Metrics works in standalone mode partially, it can talk to InfluxDB in standalone mode, but it cannot talk to Telegraf, because some shaded libraries cannot be loaded in standalone mode)

Reply:
One question: Currently, in Pravega `standalone` we have a very useful `csvReporter` that allows us to see the output of the metrics locally. This is great for development purposes. Are we going to be able of using the `csvReporter` (or something similar) after this change?

Reply:
Sorry Micrometer doesn't support CSV yet, which is probably the biggest downside of the migration unfortunately.
This could be definitely an enhancement after the migration. Micrometer currently support console output, but it's less helpful as file output.

Reply:
I'm sure after the migration is in place, teams will raise issues for improvements.